### ISSUE TITLE
Please move Npcap issues tracker to its own repository

### ISSUE THREAD

**Issue by jb-wisemo** [nmap/nmap#1756]
[ wishlist ]
Currently, Npcap, including all related code and bugreports, is mixed with the general Nmap github repository.  This makes it harder for users of either tool (only) to find code and bug reports.  This potentially affects both nmap on platforms with a native pcap library and npcap used with WireShark etc.

Thus it would be nice to separate Npcap into its own repository (perhaps github.com/nmap/npcap), moving code and bug reports accordingly.

**Comment by fyodor** [comment]
Thanks for the note, but Npcap is already in its own repository at the link you mentioned.  Only the issues database is shared.  But we use labesls in the issue tracker, so you can find and search only Npcap issues here: https://github.com/nmap/nmap/labels/Npcap

Npcap has become so popular that we might consider splitting it off into its own issue tracker rather than using labels as we do now.  I'm closing this ticket, but people can still comment as desired and we will continue to consider how to best organize the Nmap Project issue tracker.

**Comment by jb-wisemo** [comment]
Thanks, just noticed while preparing another bug report.  Please add the wishlist and npcap tags to this closed issue.

**Comment by fyodor** [comment]
Done (added labels)

**Comment by guyharris** [comment]
> Thanks for the note, but Npcap is already in its own repository at the link you mentioned. Only the issues database is shared. But we use labesls in the issue tracker, so you can find and search only Npcap issues here: https://github.com/nmap/nmap/labels/Npcap

There are a number of Npcap issues that haven't yet been labeled as such.

> Npcap has become so popular that we might consider splitting it off into its own issue tracker rather than using labels as we do now.

Having a separate tracker would help with the "Npcap issues not labeled as such" problem.  Nmap isn't the only big client of Npcap, so having a separate tracker makes sense to me.  (Having its own repository means it already has its own pull requests tab.)

**Comment by fyodor** [comment]
Yeah, we probably should move the Npcap issue tracker to its own repo rather than sharing.  And it shouldn't be too hard since we already have the separate repo and it [looks like Github supports moving individual issues between repos](https://help.github.com/en/github/managing-your-work-on-github/transferring-an-issue-to-another-repository).  I've edited the title to focus on moving the issues tracker (since we already have a separate repo) and am reopening this issue.  Cheers!

**Comment by guyharris** [comment]
"mobr"?

**Comment by fyodor** [comment]
LOL, fixed mobr -> move

**Comment by guyharris** [comment]
> LOL, fixed mobr -> move

Just one key to the left....

**Comment by fyodor** [comment]
OK I enabled issue tracking for the Npcap repo and moved all open "Npcap" labeled open issues there.  It was a bit of a pain because labels don't transfer, so I had to do those manually.  I also found and transferred a bunch which weren't properly labelled by searching the body for Npcap.  I didn't worry about already closed Npcap issues because there are several hundred of them and those aren't as important and I don't have time to deal with them now.  From now on, people should file new Npcap issues at https://github.com/nmap/npcap/issues instead of the Nmap tracker.  There is still a bit more cleanup we should do around this (like updating the docs to point to new tracker), but I am closing this issue and will open a new one for that stuff if necessary.
